src/main/java/network/brightspots/rcv/Logger.java

```diff
@@ -123,7 +123,7 @@ static void addTabulationFileLogging(String outputPath) throws IOException {
   }
 
   // function: removeTabulationFileLogging
-  // purpose: remove file logging once a tabulation run is complete
+  // purpose: remove file logging once a tabulation run is completed
```

Comment:
debatable!

Reply:
Hah, throw another one up on the board! In my view, "complete" is the state of something being full, whereas completed means a task was literally finished, which is the context here.